app/javascript/src/Types/libs/libdefs.js

```diff
@@ -32,7 +32,10 @@ declare module 'swagger-ui-react' {
 }
 
 declare module 'validate.js' {
-  declare module.exports: any;
+  declare type ERRORS = {
```

Comment:
I think this is the problem, validate function accepts more arguments and has several possible return values, take a look at `flow-typed/npm/validate.js_v0.x.x.js`